Changelogs for Digital Twin Manager Microservice 1025.0.0 (#4494)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-0-0-fixed-update-process-for-the-title-and-description-of-pre-defined-library-definitions-ctm-2775-2777.md

---

> date: ""

Reviewer: ```suggestion
date: "2026-04-09"
```

---
date: ""

> title: "Fixed update process for the title and description of pre-defined library definitions [CTM-2775] (#2777)"

Reviewer 2: ```suggestion
title: "Preserve customization of name and description of pre-defined library definitions"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2775"
version: "1025.0.0"
---

> Updates the onboarding process of library definitions so that changes to

Reviewer 3: @lara-c8y This must probably be removed as you rephrased the entry in the last paragraph?

`title` and `description` in the pre-defined definitions are applied
only when the values of the definitions in the tenant’s inventory

> haven’t been customized.

Reviewer 2: ```suggestion
During the onboarding process for library definitions, customizations made to the title and description fields were being lost when the library definitions were updated. The update process now preserves any customizations you make to the title and description fields during onboarding.
```
